src/test/java/com/accelad/docx4j/tags/TagUpdaterTableTest.java

```diff
@@ -57,4 +55,16 @@ public void should_return_true_when_updating_the_tag_is_completed() throws Excep
 
         assertThat(result, is(true));
     }
+
+    @Test
+    public void name() {
+        String[] header = { "header1", "header2" };
+        String[][] values = { { "01", "02" }, { "A", "B" } };
+        TagValueTable tagValueTable = new TagValueTable("title", header, values);
+
+        assertEquals(
+                "TagValueTable{title='title', headers=[header1, header2], headerGroups=null, tableValues=[01, 02], [A, B]}",
+                tagValueTable.toString());
+
+    }
```

Comment:
The name of the test is not very explicit... Do we need to test that? Is it the specification?
It should be formatted as the others.